array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,17 @@
+/* exported unique */
+function unique(array) {
+  var uniqueArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (uniqueArray.length === 0) {
+      uniqueArray.push(array[i]);
+    }
```

Comment:
Instead of having this conditional run on each iteration of your loop (even though it would only be relevant on the first iteration), you should add the value at the first index to your array of unique values before the loop begins, and simply begin your loop at `1` rather than `0`.